json/tbd_warehouse4.json

```diff
@@ -0,0 +1,21 @@
+{
+  "schema": "https://github.com/rd2/tbd/blob/master/tbd.schema.json",
+  "description": "test_warehouse.osm with negative PSI values",
+  "psis": [
+    {
+      "id": "BETBG Detail 7.6.2",
+      "rimjoist": 0.0,
+      "parapet": 0.0,
+      "fenestration": 0.0,
+      "corner": -0.0471,
```

Comment:
Although rare, one will come across negative PSI values in the literature, generally when dealing with efficient thermal breaks while relying on outside measurements (as per A-90.1).